routers/web/user/task.go

```diff
@@ -28,6 +30,27 @@ func TaskStatus(ctx *context.Context) {
 		return
 	}
 
+	if err := task.LoadRepo(ctx); err != nil {
+		ctx.JSON(http.StatusInternalServerError, map[string]any{
+			"err": err,
+		})
+		return
+	}
+
+	perm, err := access_model.GetUserRepoPermission(ctx, task.Repo, ctx.Doer)
+	if err != nil {
+		ctx.JSON(http.StatusInternalServerError, map[string]any{
+			"err": err,
+		})
+		return
+	}
+	if !perm.CanRead(unit.TypeCode) {
+		ctx.JSON(http.StatusForbidden, map[string]any{
```

Comment:
forbidden or not found? 🤔 
and do we have a rule about `err` or `error` in json response.

Reply:
8d067d814114b928fff969f3cca2ac49f8be0968